tests/functional/aws-node-sdk/test/bucket/get.js

```diff
@@ -342,27 +341,18 @@ describe('GET Bucket - AWS.S3.listObjects', () => {
         });
 
         tests.forEach(test => {
-            it(`should ${test.name}`, done => {
+            it(`should ${test.name}`, async () => {
                 const s3 = bucketUtil.s3;
                 const Bucket = bucketName;
-
-                Promise
-                    .mapSeries(test.objectPutParams(Bucket),
-                        param => s3.putObject(param).promise())
-                    .then(() =>
-                        s3.listObjects(test.listObjectParams(Bucket)).promise())
-                    .then(data => {
-                        const isValidResponse =
-                            tv4.validate(data, bucketSchema);
-                        if (!isValidResponse) {
-                            throw new Error(tv4.error);
-                        }
-                        return data;
-                    }).then(data => {
-                        test.assertions(data, Bucket);
-                        done();
-                    })
-                    .catch(done);
+                for (const param of test.objectPutParams(Bucket)) {
+                    await s3.putObject(param).promise();
+                }
```

Comment:
used to be done in parallel, which can be achieved still with `Promise.all()` or `Promise.allSettled()`

Reply:
Same here, about the bluebird's `mapSeries` function. The equivalent is `Promise.each`.
cf http://bluebirdjs.com/docs/api/promise.mapseries.html